docs/reference/service/configuration.md

```diff
@@ -109,8 +109,7 @@ This setting can be a `boolean` or an `object`. If set to `true` the Prometheus
 
 Supported object properties:
 
-- **`hostname`** (`string`) — The hostname where Prometheus server will listen for connections.
-- **`port`** (`number` or `string`) — The port where Prometheus server will listen for connections.
+- **`server`** (`'parent'` or `object`) — Either an object with `port` and `hostname` properties to use for the prometheus server or the string `'parent'` to run on the same port and host as platformatic itself.
```

Comment:
Should we mention that the `'parent'` config isn’t recommended by prometheus?

Reply:
yes, thx, the docs is also missing the parent option

Reply:
never mind